demos/components/src/components/VideoTile/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useRef } from 'react';
+import { StyledVideoTile } from './Styled';
+
+export interface VideoTileProps {
+  nameplate: string;
+  bindVideoTile: (arg0: any) => void;
+}
+
+export const VideoTile: React.SFC<VideoTileProps> = props => {
+  const videoEl = useRef(null);
+  const { bindVideoTile } = props;
+
+  useEffect(() => {
+    !!videoEl && bindVideoTile(videoEl.current);
+  }, [bindVideoTile, videoEl]);
```

Comment:
Storybook is being used to document and present UI components. In this case, we won't be binding a videoTile to the SDK, so I'm passing an empty function so that no action is taken.

Good catch on `useEffect`, I'll update  so that it is only called once. (The equivalent of `componentDidMount`.